src/main/java/uk/ac/ebi/ampt2d/accession/server/FileAccessionResponse.java

```diff
@@ -42,12 +45,15 @@ public void setAccessions(List<FileAccession> accessions) {
         this.accessions = accessions;
     }
 
-    class FileAccession {
+    static class FileAccession {
 
         private UuidFile file;
 
         private UUID accession;
 
+        public FileAccession() {
```

Comment:
Not Needed unless it is gonna be used for a post parsing, and in that case package protected